docs/lib/ssr/fragments/js/getting-started.md

```diff
@@ -0,0 +1,106 @@
+To work well with server-rendered pages, Amplify JS requires slight modifications from how you would use it in a client-only environment.
+
+## Amplify
+
+### Enabling SSR 
+
+When using the Amplify CLI, the __aws-exports.js__ file gets created and updated automatically for you based upon the resources you have added and configured.
+
+For client-only apps, `Amplify.configure(awsExports)` is all you need.
+
+To enable SSR support, also provide `ssr: true`:
+
+```js
+import { Amplify } from "aws-amplify"
```

Comment:
Seems like we are missing semi-colons throughout. Can be a separate PR to come back through and add them. 

```suggestion
import { Amplify } from "aws-amplify";
```